qutip/metrics.py

```diff
@@ -292,6 +292,59 @@ def bures_angle(A, B):
     return np.arccos(fidelity(A, B))
 
 
+def hellinger_dist(A, B, sparse=False, tol=0):
+    """
+    Calculates the quantum Hellinger distance between two density matrices.
+    See: D. Spehner, F. Illuminati, M. Orszag, and W. Roga, "Geometric
+    measures of quantum correlations with Bures and Hellinger distances"
+    arXiv:1611.03449
+
+    Parameters
+    ----------
+    A : qobj
```

Comment:
please change
```
A : qobj
```
to 
```
A : :class:`qutip.Qobj`
```
so that Sphinx will make this a hyperlink in the API Documentation. Change it also elsewhere when calling QuTiP's functions or classes.